eng/pipelines/npm-tasks.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none
+
+# This pipeline helps to run NPM admin tasks like remove or add tag to a released package version or deprecate a pacakge version
+# Following variables should be set at queue time to run this pipeline
+# variable name: TaskType
+# valid Options: 'AddTag', 'RemoveTag', 'DeprecatePackage'
+
+# variable name: PackageName
+# e.g. values: '@azure/storage-blob', '@azure/core-http'
+
+# variable name: 'PkgVersion'
+# e.g. values '1.0.0', '2.1.0.beta.1'
+
+# variable name: 'TagName'
+# e.g values: 'latest', 'next'
+
+parameters:
+  - name: TaskType
+    type: string
+    default: ''
```

Comment:
I suggest providing a list of values so this is easy to pick in the UI. See https://docs.microsoft.com/en-us/azure/devops/pipelines/process/runtime-parameters?view=azure-devops&tabs=script#use-parameters-in-pipelines.

Reply:
You can also give a DisplayName which can give a nicer longer name for the parameters.

Reply:
@weshaggard I liked this idea of listing values and having different display name that I am using to show e.g. values. 
Devops is showing TaskType as radio buttons with given options. Thanks for this suggestion.